tests/integration/redis-benchmark.tcl

```diff
@@ -0,0 +1,168 @@
+source tests/support/benchmark.tcl
```

Comment:
We build redis-benchmark as keydb-benchmark so this file should be renamed to match. These file names are shown to users when tests are run.